2019-May-Season/SoftUni-Information-Services/src/SIS.HTTP/Common/GlobalConstants.cs

```diff
@@ -4,6 +4,10 @@ public class GlobalConstants
     {
         public const string HttpOneProtocolFragment = "HTTP/1.1";
 
+        public const string ContentTypeHeaderKey = "Content-Type";
```

Comment:
Do header's type/key constants really belong in GlobalConstants.cs and not in another class?